### DISCUSSION TITLE
Difficulté des épreuves

### DISCUSSION THREAD

**Discussion by uurshin** [fibreville/minotaure_php#40] [Boite à idées]
Actuellement, le test des épreuves est calculé ainsi : on tire un dé de 6 et on rajoute la difficulté (de -2 à +2), si c'est en dessous ou égal à la carac le personnage réussit le test.

Ça veut dire qu'à partir d'un test difficile (+1 en difficulté), un personnage qui a 1 dans la carac échoue automatiquement.
Que pensez-vous de laissez la possibilité d'introduire une réussite critique ? Si le jet de dé fait 1, le personnage réussirait dans tous les cas ?

**Comment by PixelReveur** [comment]
Effectivement tu soulèves un bon point, ce serait bien de laisser une possibilité de réussir (et une possibilité d'echec pour les persos qui ont plus de 6). Ceci dit, une chance sur 6 de faire un critique ça me semble quand même beaucoup... Sur un jet de difficulté +2, les personnages qui ont 1 dans la carac auraient du coup autant de chances de réussir qu'un personnage qui a 3 dans la carac, ça me semble un peu injuste pour les personnages qui ont 3 dans la carac.

Peut être qu'en plus du d6, tous les personnages pourraient jeter un d20. Un 1 sur ce dé serait une réussite automatique, un 20 serait un échec automatique.

**Comment by uurshin** [comment]
Très bonne idée le d20, je vais rajouter ça !

**Comment by ElethornGM** [comment]
Tant qu'à parler des épreuves, je trouve que les caracs 1 à 6 sont pas super en terme de flexibilité, parce que dès que l'on donne un objet qui donne +1 ou +2 dans une caractéristiques, la modif est très énorme.
Par exemple, quelqu'un qui a 5 de corps et qui obtient un objet +1 en corps, c'est assez fou...
ça pourrait être intéressant sinon de passer les caractéristiques sur 20 ou sur 100, pour pouvoir avoir des malus/bonus plus variés et pouvoir donner des objets qui n'ont pas un impact aussi important.

Juste une idée comme ça.

**Comment by uurshin** [comment]
Il faut bien voir que le principe de base de tout ATRGP est justement la simplification pour en fait un jeu "macroscopique" et volontairement très schématique sur beaucoup de choses, au delà de la difficulté technique de faire autrement je pense que c'est aussi une vraie volonté de la part de FT. Ca a du sens quand on sait que ses streams réunissent plusieurs centaines de personnes.

Ceci étant dit, on peut très bien changer ce principe et adopter du D20 (je préfère au D100, ça fera un caractère de moins par perso) !
Il faudrait alors peut être proposer des difficultés plus granulaires (de 1 à 20), peut être à l'aide d'un slider (https://refreshless.com/nouislider/pips/) ?

On pourrait également choisir de rendre ça paramétrable par partie (D6, D20, D100) mais c'est plus de boulot de dev.

**Reply by ElethornGM** [reply]
Je comprends aussi le point de vue de l'outil simple... C'est pour ça que je propose, sans trop savoir ce que le reste de la commu en penserais. C'est sûr qu'en l'était actuel, je trouve que les "loots" ne sont pas très maniables (donne tout de suite trop de bonus), et perso je ne pense pas les utiliser beaucoup.
Après, vu que c'est du boulot de dev, ce n'est peut être pas la priorité de toute façon.
